### PR TITLE
Show cricket score for Australian & Indian matches

### PR DESCRIPTION
**Pull request by gklopper**
We used to just show England matches...

cricket1
cricket2
cricket3

### CODE REVIEW

**Comment by gklopper** [comment]
cc @rich-nguyen

**Comment by rich-nguyen** [comment]
:+1:

File: common/app/conf/switches/PerformanceSwitches.scala [hunk 1]
```diff
@@ -38,7 +38,7 @@ trait PerformanceSwitches {
     "soft-purge-switch",
     "If this switch is on then articles will be automatically soft purged them from the CDN",
     safeState = Off,
-    sellByDate = new LocalDate(2015, 12, 15),
+    sellByDate = never,
```
Comment (gklopper) [inline common/app/conf/switches/PerformanceSwitches.scala]: Not directly related to this PR, but "while I am in here"

File: common/app/conf/switches/PerformanceSwitches.scala [hunk 2]
```diff
@@ -47,7 +47,7 @@ trait PerformanceSwitches {
     "long-cache-switch",
     "If this switch is on then articles will get a longer cache time",
     safeState = Off,
-    sellByDate = new LocalDate(2015, 12, 15),
+    sellByDate = never,
```
Comment (gklopper) [inline common/app/conf/switches/PerformanceSwitches.scala]: Not directly related to this PR, but "while I am in here"

**Pull request merged**